docs/docs/migration/migration-from-v5-to-v6.md

```diff
@@ -116,13 +128,6 @@ It automatically installs the new versions of all `@comet` libraries and handles
 
 6. Rename and add decorators
```

Comment:
This should be something like "Adapt decorators"

Reply:
Done